modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/components/_activation-keys.scss

```diff
@@ -41,11 +41,15 @@
 .cp-activation-key-icon {
 	border: 1px solid var(--color-neutral-2);
 	box-shadow: none;
-	left: 641px;
+	left: 648px !important;
 	min-width: fit-content;
-	top: 276px;
-	ul li span {
+	top: 276px !important;
+	ul li button {
 		padding: 0.75rem;
 		padding-right: 68px !important;
 	}
 }
+
+.cp-activation-status-container .card-body {
+	padding-right: 13px !important;
```

Comment:
Do not use paddings/margins in CSS customization, our pattern is to always use even numbers. So I think is a better solution to use className="pr-3"(16px) OR pr-2 (8px) + pr-1 (4px) (two different elements)

Reply:
Thanks @iagochaves . Done!